lib/timescaledb/migrations/v01.ex

```diff
@@ -0,0 +1,51 @@
+defmodule Membrane.RTC.Engine.TimescaleDB.Migrations.V01 do
+  @moduledoc false
+
+  use Ecto.Migration
+
+  @spec up() :: any()
+  def up() do
+    create table(:peers_metrics, primary_key: {:id, :id, autogenerate: true}) do
+      add(:peer_id, :string, null: false)
+      add(:room_id, :string, null: false)
+      add(:"ice.binding_requests_received", :integer)
+      add(:"ice.binding_responses_sent", :integer)
+      add(:"ice.bytes_received", :integer)
+      add(:"ice.bytes_sent", :integer)
+      add(:"ice.packets_received", :integer)
+      add(:"ice.packets_sent", :integer)
+
+      timestamps(type: :utc_datetime_usec, updated_at: false)
+    end
+
+    create(index(:peers_metrics, [:inserted_at]))
+    create(index(:peers_metrics, [:peer_id]))
+    create(index(:peers_metrics, [:inserted_at, :peer_id]))
+
+    flush()
+
+    create table(:tracks_metrics, primary_key: {:id, :id, autogenerate: true}) do
+      add(:track_id, :string, null: false)
+      add(:peer_metrics_id, references(:peers_metrics))
```

Comment:
I guess you should configure the on_delete & on_update https://hexdocs.pm/ecto_sql/Ecto.Migration.html#references/2